build/changelog-generator/templates/header.hbs

```diff
@@ -13,4 +13,4 @@
 
 
 ~~ TODO: https://chromiumdash.appspot.com/schedule ~~
-We expect this release to ship in the DevTools of Chrome XX.
+We expect this release to ship in the DevTools of [Chrome XX](https://chromiumdash.appspot.com/schedule).
```

Comment:
yea?

Reply:
theres this https://calendar.google.com/calendar/embed?src=google.com_c1f795f7cg83sv2qu15c4v1kfk%40group.calendar.google.com

not very automatable